index.html

```diff
@@ -8,5 +8,6 @@
   </head>
   <body>
     uccii_test_5th
+    <p>第１回 はじめてのgit講座</p>
```

Comment:
p を div にしてください。
なぜなら、、、、
```suggestion
    <div>第１回 はじめてのgit講座</div>
```